stats/view/worker_commands.go

```diff
@@ -57,6 +58,7 @@ func (cmd *subscribeToViewReq) handleCommand(w *worker) {
 			errstr = append(errstr, fmt.Sprintf("%s: %v", view.Name, err))
 			continue
 		}
+		internal.SubscriptionReporter(view.Measure.Name())
```

Comment:
may as well just pass Measure and call the .subscribe directly

Reply:
I can't, it creates a circular dependency.